src/libm-tester/tester.c

```diff
@@ -4348,6 +4348,7 @@ void do_test() {
     for(y = -1e+7;y < 1e+7 && success;y += 1.51e+5) {
       for(x = -1e+7;x < 1e+7 && success;x += 1.51e+5) checkAccuracy_f_f(mpfr_remainder, child_remainderf, y, x, 0.5);
     }
+    checkAccuracy_f_f(mpfr_remainder, child_remainderf, 11114942644092928.0, 224544296009728.0, 0.5);
```

Comment:
I think you want to add a test also for the double precision case.

Reply:
Only single precision cases are found.

Reply:
I see - if that's the case, I think you should revert the change in the double precision implementation.

Reply:
No. Those two implementations work with the same principle. So, if the single-precision version fails, there should be cases for the double-precision version to fail.